dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/config/WSKeyStore.java

```diff
@@ -1045,7 +1089,7 @@ public void store() throws Exception {
             boolean fileBased = Boolean.parseBoolean(getProperty(Constants.SSLPROP_KEY_STORE_FILE_BASED));
             String SSLKeyStoreStash = getProperty(Constants.SSLPROP_KEY_STORE_CREATE_CMS_STASH);
 
-            KeyStore ks = getKeyStore(false, false);
+            KeyStore ks = clonedKeyStore == null ? getKeyStore(false, false) : clonedKeyStore;
```

Comment:
See other findings about setting the keystore in this method. Only need to do it if we sent in the clonedKeyStore.